main.go

```diff
@@ -23,10 +23,11 @@ var ResourceVersion = "dev"
 
 type Options struct {
 	Telemetry struct {
-		Host     string `long:"host" description:"the url of the host to receive the telemetry (or honeycomb, dogfood, local)" default:"honeycomb"`
-		Insecure bool   `long:"insecure" description:"use this for insecure http (not https) connections" yaml:",omitempty"`
-		Dataset  string `long:"dataset" description:"sends all traces to the given dataset" env:"HONEYCOMB_DATASET" default:"loadgen"`
-		APIKey   string `long:"apikey" description:"the honeycomb API key(*)" env:"HONEYCOMB_API_KEY" yaml:"-"`
+		Host     string   `long:"host" description:"the url of the host to receive the telemetry (or honeycomb, dogfood, local)" default:"honeycomb"`
+		Insecure bool     `long:"insecure" description:"use this for insecure http (not https) connections" yaml:",omitempty"`
+		Dataset  string   `long:"dataset" description:"sends all traces to the given dataset" env:"HONEYCOMB_DATASET" default:"loadgen"`
+		Services []string `long:"services" description:"list of services to send traces as. Only works with otel" env:"HONEYCOMB_SERVICES" default:""`
```

Comment:
Maybe we can have a default value for the services list? It's more convenient if users don't care about the number of services in their data